src/Sylius/Component/Order/Model/Order.php

```diff
@@ -491,7 +504,7 @@ public function getAdjustmentsTotal($type = null)
         }
 
         $total = 0;
-        foreach ($this->getAdjustments($type) as $adjustment) {
+        foreach ($this->getAdjustmentsRecursively($type) as $adjustment) {
```

Comment:
Yep, I was wondering why is it here and I've just realized that I changed this but in other PR :D Surely it should be there.